master-galera/master.cfg

```diff
@@ -390,6 +390,8 @@ for os_i in os_info:
         if os_i == "ubuntu-2004":
             print("using gcc/++-10")
             env = {"CC": "gcc-10", "CXX": "g++-10"}
+        if worker_name.startswith("ppc64le-db"):
+            env = {"JOBS": "16"}
 
```

Comment:
Why is this an issue only for the db machine? Does the galera build use the JOBS env var?

Reply:
It detects 128 cores and tries to use all of them and the build shows there's insufficent memory.

I don't know how much memory the Raptor system has in comparison, or other systems on a memory/per core basis so other systems of a comparable ratio might need it.

Yes, galera build uses JOBS env variable.